src/angularjs/src/app/components/map/map.constants.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+  'use strict';
+
+  var config = {
+    baseLayers: {
+        'Stamen' : {
+            url: 'https://stamen-tiles.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.png',
+            attribution: ['Map tiles by <a href="http://stamen.com">Stamen Design</a>, ',
+                'under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. ',
+                'Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under ',
+                '<a href="http://www.openstreetmap.org/copyright">ODbL</a>.'
+            ].join('')
+        },
+        'Satellite': {
+            url: 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
+            attribution: [
+                '&copy; <a href="http://www.esri.com/">Esri</a> ',
+                'Source: Esri, DigitalGlobe, GeoEye, Earthstar Geographics, CNES/Airbus DS, USDA, USGS, ',
+                'AEX, Getmapping, Aerogrid, IGN, IGP, swisstopo, and the GIS User Community'
+            ].join('')
+        }
+    },
+    conusBounds: [[24.396308, -124.848974], [49.384358, -66.885444]],
+    conusMaxZoom: 18
```

Comment:
Can you drop this to 17 since that's the highest zoom we'll be generating via tiles?

See https://github.com/azavea/pfb-network-connectivity/pull/364#discussion_r113988746